support/index.adoc

```diff
@@ -12,10 +12,20 @@ Red Hat offers cluster administrators tools for gathering data for your cluster,
 == Get support
 xref:../support/getting-support.adoc#getting-support[Get support]: Visit the Red Hat Customer Portal to review knowledge base articles, submit a support case, and review additional product documentation and resources.
 
+ifndef::openshift-rosa,openshift-dedicated[]
```

Comment:
Just checking - should I be seeing this section in OSD and ROSA versions? I think the intent is for it to _not_ show, but I am seeing it there. I could be confused though.

- https://63305--docspreview.netlify.app/openshift-dedicated/latest/support/#support-overview-remote-health-monitoring
- https://63305--docspreview.netlify.app/openshift-rosa/latest/support/#support-overview-remote-health-monitoring